rf-stylez.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = ['rf-stylez']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rubocop', '~> 1.1.0'
+  spec.add_runtime_dependency 'rubocop', '>= 1.1', '< 1.12'
```

Comment:
Since we install `rf-stylez` directly now I think it's more useful to pin rubocop version directly (like `rubocop-rails` below).

We're probably using range here because we used to add rf-stylez to :allthegemfiles:

Reply:
And the reasoning behind why I think it's more useful is we want specific version of `rf-stylez` to guarantee a specific version of `rubocop` - now it'll be anything between `1.1` and `1.12`

Reply:
If we're pinning dependencies we should add a comment with the rationale in the Gemfile (or in the commit message) so it's clear why we've done it as close as possible to the change.

Reply:
IMO If we pin versions we should add the reason why to the Gemfile (or the commit message) so it's obvious why the pin exists.